mkt/extensions/models.py

```diff
@@ -75,6 +75,7 @@ class Extension(ModelBase):
 
     # Fields for which the manifest is the source of truth - can't be
     # overridden by the API.
+    author = models.CharField(default='', editable=False, max_length=128)
```

Comment:
I try to avoid it for `CharField`s for new code when I can, because of https://docs.djangoproject.com/en/1.8/ref/models/fields/#null